nixos/modules/programs/flashprog.nix

```diff
@@ -0,0 +1,28 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.programs.flashprog;
+in
+{
+  options.programs.flashprog = {
+    enable = lib.mkOption {
+      type = lib.types.bool;
+      default = false;
+      description = ''
+        Installs flashprog and configures udev rules for programmers
+        used by flashprog.
+      '';
+    };
+    package = lib.mkPackageOption pkgs "flashprog" { };
+  };
+
+  config = lib.mkIf cfg.enable {
+    services.udev.packages = [ cfg.package ];
```

Comment:
Since the libjaylink package brings its own rules, should we introduce a dedicated module or is it appropriate to add libjaylink here?

Reply:
@bjornfor Do you have an opinion?

Reply:
I don't have an opinion on this :-)

Reply:
I think the proper way to do this is with dedicated NixOS modules. I will leave this as for now and update it later.

Reply:
To clarify, do you want me to merge this PR as-is now?

Reply:
Yes, please :)